akka-cluster-tools/src/main/scala/akka/cluster/singleton/ClusterSingletonManager.scala

```diff
@@ -901,6 +1016,21 @@ class ClusterSingletonManager(singletonProps: Props, terminationMessage: Any, se
       if (m.uniqueAddress == cluster.selfUniqueAddress)
         logInfo("Self downed, waiting for removal")
       stay
+    case Event(ReleaseLeaseFailure(t), _) =>
+      // TODO we could retry
```

Comment:
Should we? If so ticket? If not remove comment?

Reply:
i'll remove for now, we can see if it happens enough to justify retries and not just waiting for the timeout